sdk/identity/azure-identity/tests/recorded_test_case.py

```diff
@@ -4,35 +4,20 @@
 # ------------------------------------
 import os
 
-from azure_devtools.scenario_tests import GeneralNameReplacer, patch_time_sleep_api, RequestUrlNormalizer
-from devtools_testutils.azure_testcase import AzureTestCase
+from devtools_testutils import AzureRecordedTestCase, is_live
 import pytest
 
-from recording_processors import IdTokenProcessor, RecordingRedactor
-
 PLAYBACK_CLIENT_ID = "client-id"
 
 
-class RecordedTestCase(AzureTestCase):
-    def __init__(self, *args, **kwargs):
-        scrubber = GeneralNameReplacer()
-        super(RecordedTestCase, self).__init__(
-            *args,
-            recording_processors=[RecordingRedactor(), scrubber],
-            replay_processors=[IdTokenProcessor(), RequestUrlNormalizer()],
-            **kwargs
-        )
-        self.scrubber = scrubber
-        self.replay_patches.append(patch_time_sleep_api)
```

Comment:
To replicate the past use of `patch_time_sleep_api`, we could probably add `patch_sleep` and `patch_async_sleep` (if necessary) methods to `conftest.py`, copying Kashif's implementation for KV: https://github.com/Azure/azure-sdk-for-python/blob/e769b81a996db0b4a4ccd72773425fe31e53c518/sdk/keyvault/azure-keyvault-secrets/tests/conftest.py#L58-L68